droid-configs.inc

```diff
@@ -59,6 +59,12 @@
 %define rpm_vendor %{vendor}
 %endif
 
+%if 0%{?android_flavour:1}
```

Comment:
Android flavour is a vague term: if someone wants to refactor Android 8, they won't know whether to write `p` or `pie`, and flavours are dropped in favour of numbered versions starting with 10.

We already have android version handling in dhd, I suggest to align with that and use `android_version_major` (always numbers).
The only difference is that here in dcd it will be specified manually, and in dhd we have the means to autodetect it.

Reply:
> The only difference is that here in dcd it will be specified manually, and in dhd we have the means to autodetect it.
That is true currently but maybe once this it done we can reverse this and make it opt out.